web/api/mocks/data/steps.json

```diff
@@ -8,14 +8,11 @@
     "targetDatabase": "staging",
     "isValid": true,
     "options": {
-      "input_file_path": ".",
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.